requirements/requirements-braket.txt

```diff
@@ -0,0 +1,2 @@
+amazon-braket-sdk~=1.69.0
+cirq-ionq>=1.0.0,<1.4.0
```

Comment:
Why is `cirq-ionq` required for the `braket` integration?

Reply:
Because of this: https://github.com/unitaryfund/mitiq/blob/8c24a9bddd4d6213cdd39fae2d750e55247e82a9/mitiq/interface/mitiq_braket/conversions.py#L8